py/picca/delta_extraction/data_catalogues/desi_data.py

```diff
@@ -307,35 +310,33 @@ def read_from_minisv_desi(self, catalogue):
             try:
                 hdul = fitsio.FITS(filename)
             except IOError:
-                logging.warning(f"Error reading file {filename}. Ignoring file")
+                self.logger.warning(f"Error reading file {filename}. Ignoring file")
                 continue
 
             fibermap = hdul['FIBERMAP'].read()
             fibermap_colnames = hdul["FIBERMAP"].get_colnames()
-            # pre-Andes
-            if 'TARGET_RA' in fibermap_colnames:
-                ra = fibermap['TARGET_RA']
-                dec = fibermap['TARGET_DEC']
-                tile_spec = fibermap['TILEID'][0]
-                night_spec = fibermap['NIGHT'][0]
+            # SV releases
+            ra_spec = fibermap['TARGET_RA']
+            dec_spec = fibermap['TARGET_DEC']
+
+            tile_spec = fibermap['TILEID'][0]
+            night_spec = fibermap[nightcol][0]
+            try:
+                hdul['BRZ_WAVELENGTH']
+            except OSError:
+                colors = ['B','R','Z']
+                if index == 0:
+                    self.logger.warning(
+                        "Reading single band coadds, picca will coadd the bands,"
+                        "as typical since SV")
+            else:
                 colors = ['BRZ']
                 if index == 0:
-                    logging.warning(
+                    self.logger.warning(
                         "Reading all-band coadd as in minisv pre-Andes "
                         "dataset")
-            # Andes
-            elif 'RA_TARGET' in fibermap_colnames:
-                ra = fibermap['RA_TARGET']
-                dec = fibermap['DEC_TARGET']
-                tile_spec = filename.split('-')[-2]
-                night_spec = int(filename.split('-')[-1].split('.')[0])
-                colors = ['B', 'R', 'Z']
-                if index == 0:
-                    logging.warning(
-                        "Couldn't read the all band-coadd, trying "
-                        "single band as introduced in Andes reduction")
-            ra = np.radians(ra)
-            dec = np.radians(dec)
+            ra_spec = np.radians(ra_spec)
+            dec_spec = np.radians(dec_spec)
```

Comment:
We are not using ra_spec and dec_spec for anything, aren't we? Or did I miss something? Maybe we can just delete lines 319, 320, 321, 338, and 339

Reply:
These lines are removed in commit #cc64256